scripts/Iguana-Tweaks.zs

```diff
@@ -24,6 +24,8 @@ recipes.remove(<IguanaTweaksTConstruct:sawManyullyn>);
 
 
 // --- Unfired Clay Bucket
+// If you change this recipe, be sure to update gregtech.api.enums.OrePrefixes!
+// bucketClay.mSecondaryMaterial controls what the clay bucket pulverizes into.
 recipes.addShaped(<IguanaTweaksTConstruct:clayBucketUnfired>, [
 [null, null, null],
 [<ore:dustClay>, null, <ore:dustClay>],
```

Comment:
This recipe seems to be wrong or not used.

![image](https://user-images.githubusercontent.com/2970688/127648753-053202fb-ce0b-4095-b0c7-a3edfa66b5c2.png)
 This is the one we use

Reply:
Just unwrap one line below in github GUI

Reply:
Isn't that the same as the one in this file? 5x clay dust:
```
[null, null, null],
[<ore:dustClay>, null, <ore:dustClay>],
[<ore:dustClay>, <ore:dustClay>, <ore:dustClay>]]);
```